Description: Update dir for installing bash completions
 Debian uses /usr/share/bash-completion/completions
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2017-09-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.am
+++ b/Makefile.am
@@ -138,7 +138,7 @@
 systemdsystemunitdir = $(prefix)/lib/systemd/system
 dist_systemdsystemunit_DATA = lldpad.service lldpad.socket
 
-bashcompletiondir = $(sysconfdir)/bash_completion.d
+bashcompletiondir = $(prefix)/usr/share/bash-completion/completions
 dist_bashcompletion_DATA = contrib/bash_completion/lldpad contrib/bash_completion/lldptool
 
 AM_DISTCHECK_CONFIGURE_FLAGS = --enable-debug
